ftlib/consensus/gossip/memberlist/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+    /*
+       #include <stdlib.h>
+       typedef struct member_list{
+           char *addrs[1024];
+           int size;
+       } member_list;
+    */
+    "C"
+    "fmt"
+    "github.com/hashicorp/memberlist"
+    "os"
+)
+
+var list *memberlist.Memberlist
+
+
+func main() {
```

Comment:
Remove this if not used?

Reply:
I believe it is necessary to keep a main function in the main package.